### ISSUE TITLE
[BUG] Always failed in CircleCI

### ISSUE THREAD

**Issue by hideokamoto** [microsoft/playwright#3871]
**Context:**
- Playwright Version: 1.3.0
- Operating System: Docker (mcr.microsoft.com/playwright:next)
- Node.js version: Docker (mcr.microsoft.com/playwright:next)
- Browser: Docker (mcr.microsoft.com/playwright:next)

**Describe the bug**

We always get an error by it in `yarn install`.
We never test and deploy our app after installing it.

<img width="1471" alt="スクリーンショット 2020-09-14 16 32 47" src="https://user-images.githubusercontent.com/6883571/93056935-170eb580-f6a8-11ea-9144-c5e47ff75094.png">

```bash
$ yarn --frozen-lockfile
yarn install v1.22.5

[4/4] Building fresh packages...
error /root/workspace/node_modules/playwright: Command failed.
Exit code: 1
Command: node install.js
Arguments: 
Directory: /root/workspace/node_modules/playwright
Output:
Failed to process descriptor at /tmp/pw/node_modules/playwright
Removing unused browser at /root/.cache/ms-playwright/chromium-799411
Removing unused browser at /root/.cache/ms-playwright/firefox-1173
Removing unused browser at /root/.cache/ms-playwright/webkit-1344

(node:101) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, chmod '/root/.cache/ms-playwright/chromium-792639/chrome-linux/chrome'
(node:101) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:101) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

Exited with code exit status 1
```

## .circleci/config.yml

```yaml
# Javascript Node CircleCI 2.0 configuration file
#
# Check https://circleci.com/docs/2.0/language-javascript/ for more details
#
version: 2.1
workflows:
  version: 2
  build:
    jobs:
      - test-and-lint
      - e2e-test

executors:
  e2e:
    docker:
      - image: mcr.microsoft.com/playwright:next
        environment:
          NODE_ENV: development 
    working_directory: ~/workspace

jobs:
  e2e-test:
    executor: e2e
    steps:
      - checkout
      - run: yarn --frozen-lockfile
      - run:
          name: build development application
          command: REACT_APP_API_MODE=development yarn build
      - run:
          name: E2E test
          command: yarn test:e2e:ci
```

**Comment by aomarks** [comment]
Seeing a similar error on GitHub Actions:

```
> node install.js


lerna ERR! npm ci stderr:

(node:7990) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, chmod '/home/runner/.cache/ms-playwright/chromium-792639/chrome-linux/chrome'
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7990) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7990) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! playwright@1.3.0 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the playwright@1.3.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Using this config:

```
name: Tests

on: [push, pull_request]

jobs:
  tests:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2

      - uses: actions/setup-node@v1
        with:
          node-version: 14

      - uses: microsoft/playwright-github-action@v1

      - name: NPM install
        run: npm ci

      - name: Lerna bootstrap
        run: npm run bootstrap

      - name: Build
        run: npm run build

      - name: Test
        run: npm run test
```

**Comment by aslushnikov** [comment]
@aomarks I see you use Lerna to bootstrap the project. This seems to be a dupe of #3912, which was fixed and the fix released back in v1.5. Can you please check if it works for you?

**Comment by pavelfeldman** [comment]
Closing as per above, please file it and link to this issue if it repros on 1.6!

**Comment by obataku** [comment]
> @aomarks I see you use Lerna to bootstrap the project. This seems to be a dupe of #3912, which was fixed and the fix released back in v1.5. Can you please check if it works for you?

fwiw, the error here is raised on `chmod` not `symlink` and occurs during `yarn install`, not `lerna bootstrap`
